hex/lib/dependabot/hex/file_updater/mixfile_sanitizer.rb

```diff
@@ -11,16 +11,40 @@ def initialize(mixfile_content:)
           @mixfile_content = mixfile_content
         end
 
+        FILE_READ      = /File.read\(.*?\)/.freeze
+        FILE_READ_BANG = /File.read!\(.*?\)/.freeze
```

Comment:
question (non-blocking): it looks like we're trying to match `File.read`. Do we also need to match `IO.read` and/or `Pathname#read`?

Reply:
We shouldn't. These are matching Elixir functions what share a name with Ruby methods. They _do_ have a very familiar feel as the language's creator came out of the Ruby community 😄